x-pack/plugins/security_solution/cypress/tasks/api_calls/elasticsearch.ts

```diff
@@ -16,6 +15,55 @@ export const deleteIndex = (index: string) => {
   });
 };
 
+export const createIndex = (indexName: string, properties: Record<string, unknown>) =>
+  cy.request({
+    method: 'PUT',
+    url: `${Cypress.env('ELASTICSEARCH_URL')}/${indexName}`,
+    headers: { 'kbn-xsrf': 'cypress-creds' },
+    body: {
+      mappings: {
+        properties,
+      },
+    },
+  });
+
+export const indexDocument = (indexName: string, document: Record<string, unknown> = {}) =>
```

Comment:
nit: Tests should be readable and transparent without checking out what a function accepts. I'd recommend to make `document` a required parameter as it will be clear what kind of document is indexed.